tests/nfs/install.pm

```diff
@@ -46,7 +46,7 @@ sub install_dependencies_cthon04 {
 sub install_testsuite {
     my $testsuite = shift;
     if (get_var("PYNFS")) {
-        my $url = get_var('PYNFS_GIT_URL', 'git://git.linux-nfs.org/projects/bfields/pynfs.git');
+        my $url = get_var('PYNFS_GIT_URL', 'git://git.linux-nfs.org/projects/cdmackay/pynfs.git');
```

Comment:
Why not just change variable?

Reply:
IMHO we should keep the latest usable git in the sources. But, more importantly, we don't set this variable in test suites (it would be on 4 places). This variable is only for testing.

Reply:
ok, lets have latest valid source hardcoded